src/esm_parser/esm_parser.py

```diff
@@ -3022,6 +3024,43 @@ def _config_init(self, user_config):
         # pprint_config(self.config)
         # sys.exit(0)
 
+    def check_user_defined_versions(self, user_config, setup_config):
+        """
+        When running a standalone model, checks whether the users has define the
+        variable ``version`` in more than one section and if that's the case
+        throws and error. If ``version`` is only defined in the ``general`` section
+        it creates a ``version`` with the same value in the model section, ensuring
+        that the user can arbitrarily define ``version`` in either ``general`` or
+        ``<model>`` sections.
+
+        Parameters
+        ----------
+        user_config : dict
+            Experiment configuration defined by the user (e.g. runscript)
+        setup_config : dict
+            Experiment configuration defined by the default ESM-Tools configuration
+            files (``<PATH>/esm_tools/configs/``)
+
+        Raises
```

Comment:
Maybe not so important but `Raises` is used when the code explicity raises an Exception via the `raise` statement.

Reply:
Yeah, I was aware of that, however, I did not know where to put it otherwise. Do you have a suggestion?

Reply:
NumPy docstring has an optional `Notes` section: https://numpydoc.readthedocs.io/en/latest/example.html#example
Maybe that could be the place.

Reply:
+1 for notes